docs/UserGuide.md

```diff
@@ -194,21 +194,21 @@ the constraints of each parameter when used in a command.
+| `email`            | email address of the patient.                                                                              | {::nomarkdown}<ul><li>Should be in the format {:/}`local-part@domain`{::nomarkdown}.</li><li>Should not be blank.</li><li>The {:/}`local-part`{::nomarkdown} should only contain alphanumeric characters and these special characters: {:/}`+_.-`{::nomarkdown}.</li><li>The {:/}`local-part`{::nomarkdown} may not start or end with any special characters.</li><li>The {:/}`local-part`{::nomarkdown} may not contain consecutive special characters.</li><li>The {:/}`domain`{::nomarkdown} must be at least 2 characters long.</li><li>The {:/}`domain`{::nomarkdown} must start and end with alphanumeric characters.</li><li>The {:/}`domain`{::nomarkdown} cannot be the 3-character sequence: {:/}`alphanumeric`{::nomarkdown}, {:/}`-`{::nomarkdown}, {:/}`alphanumeric`{::nomarkdown}.</li><li>The {:/}`domain`{::nomarkdown}, if delimited with dots, cannot have the final dot proceeded by the 3-character sequence: {:/}`alphanumeric`{::nomarkdown}, {:/}`-`{::nomarkdown}, {:/}`alphanumeric`{::nomarkdown}.</li></ul>{:/} | *:white_check_mark: `raj@gmail.com`*{: style="white-space:nowrap"} <br>*:white_check_mark: `jie.lun+work@n.abc-123.org`*{: style="white-space:nowrap"} <br>*:x: `raj`*{: style="white-space:nowrap"} <br>*:x: `jie..work@hotmail.com`*{: style="white-space:nowrap"} <br>*:x: `yz@u.example.c-m`*{: style="white-space:nowrap"} <br> *:x: `wn@s-g`*{: style="white-space:nowrap"} |
```

Comment:
The parameter name should be in all caps: `EMAIL` and the description should start with a capital letter: 'Email address of the patient'